polygeist/lib/Dialect/Polygeist/Transforms/LoopInternalization.cpp

```diff
@@ -124,11 +177,34 @@ unsigned getLocalMemoryRemain(gpu::GPUModuleOp &module,
   return localMemoryRemain;
 }
 
-/// Get the required local memory for \p accTy.
+/// Get the required local memory for \p accTy in bytes.
+std::variant<Value, unsigned>
+getReqdLocalMemory(sycl::AccessorType accTy, const WorkGroupSize &workGroupSize,
```

Comment:
May this function belong to the `WorkGroupSize` class ?

Reply:
I also considered that, but local memory is not work group size, local memory uses work group size.